spec/blob_spec.rb

```diff
@@ -63,7 +66,62 @@
 
       it { blob.name.should == "テスト.txt" }
       it { blob.data.should include("これはテスト") }
-      it { blob.size.should == 340  }
+      it { blob.size.should == 340 }
+      it { blob.mode.should == 0100755 }
+    end
+  end
+
+  describe 'mode' do
+    context 'file regular' do
+      let(:blob) { Gitlab::Git::Blob.find(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.